UnitTests/View/ViewTests.cs

```diff
@@ -1415,7 +1429,131 @@ public void Dispose_View ()
 			Assert.Null (view.Margin);
 			Assert.Null (view.Border);
 			Assert.Null (view.Padding);
+		}
+
+		[Theory, AutoInitShutdown]
+		[InlineData (true)]
+		[InlineData (false)]
+		public void View_Draw_Horizontal_Simple_TextAlignments (bool autoSize)
+		{
+			var text = "Hello World";
+			var width = 20;
+			var lblLeft = new View (text) { Width = width, AutoSize = autoSize };
+			var lblCenter = new View (text) { Y = 1, Width = width, TextAlignment = TextAlignment.Centered, AutoSize = autoSize };
+			var lblRight = new View (text) { Y = 2, Width = width, TextAlignment = TextAlignment.Right, AutoSize = autoSize };
+			var lblJust = new View (text) { Y = 3, Width = width, TextAlignment = TextAlignment.Justified, AutoSize = autoSize };
+			var frame = new FrameView () { Width = Dim.Fill (), Height = Dim.Fill () };
+			frame.Add (lblLeft, lblCenter, lblRight, lblJust);
+			Application.Top.Add (frame);
+			Application.Begin (Application.Top);
+			((FakeDriver)Application.Driver).SetBufferSize (width + 2, 6);
+
+			Assert.True (lblLeft.AutoSize == autoSize);
+			Assert.True (lblCenter.AutoSize == autoSize);
+			Assert.True (lblRight.AutoSize == autoSize);
+			Assert.True (lblJust.AutoSize == autoSize);
+			Assert.True (lblLeft.TextFormatter.AutoSize == autoSize);
+			Assert.True (lblCenter.TextFormatter.AutoSize == autoSize);
+			Assert.True (lblRight.TextFormatter.AutoSize == autoSize);
+			Assert.True (lblJust.TextFormatter.AutoSize == autoSize);
+			Assert.Equal (new Rect (0, 0, width, 1), lblLeft.Frame);
+			Assert.Equal (new Rect (0, 1, width, 1), lblCenter.Frame);
+			Assert.Equal (new Rect (0, 2, width, 1), lblRight.Frame);
+			Assert.Equal (new Rect (0, 3, width, 1), lblJust.Frame);
+			if (autoSize) {
+				Assert.Equal (new Size (11, 1), lblLeft.TextFormatter.Size);
+				Assert.Equal (new Size (11, 1), lblCenter.TextFormatter.Size);
+				Assert.Equal (new Size (11, 1), lblRight.TextFormatter.Size);
+			} else {
+				Assert.Equal (new Size (width, 1), lblLeft.TextFormatter.Size);
+				Assert.Equal (new Size (width, 1), lblCenter.TextFormatter.Size);
+				Assert.Equal (new Size (width, 1), lblRight.TextFormatter.Size);
+			}
+			Assert.Equal (new Size (width, 1), lblJust.TextFormatter.Size);
+			Assert.Equal (new Rect (0, 0, width + 2, 6), frame.Frame);
+
+			var expected = @"
+┌────────────────────┐
+│Hello World         │
+│    Hello World     │
+│         Hello World│
+│Hello          World│
+└────────────────────┘
+";
+
+			var pos = TestHelpers.AssertDriverContentsWithFrameAre (expected, output);
+			Assert.Equal (new Rect (0, 0, width + 2, 6), pos);
+		}
 
+		[Theory, AutoInitShutdown]
+		[InlineData (true)]
+		[InlineData (false)]
+		public void View_Draw_Vertical_Simple_TextAlignments (bool autoSize)
```

Comment:
@tig this test you can't successful reproduce on your PR #3130.

Reply:
I don't know what you mean. I moved that test to #3130 and it passes.

![image](https://github.com/gui-cs/Terminal.Gui/assets/585482/0cce5e4b-f678-4f5c-aa32-2a5d2e9e15d3)

Reply:
Ops I didn't pull, sorry. Confirmed thanks.